public/locales/en/translation.json

```diff
@@ -0,0 +1,138 @@
+{
+  "commoncomponents": {
+    "button": {
+      "change": {
+        "language": "Change language"
+      },
+      "login": {
+        "dev": {
+          "account": {
+            "login": "Login with Dev account"
+          }
+        },
+        "msal": {
+          "login": "Login with Active Directory (MSAL)"
+        },
+        "regular": {
+          "login": "Sign In"
+        },
+        "static": {
+          "webapp": {
+            "login": "Login with Active Directory (Static Web App)"
+          }
+        }
+      },
+      "run": {
+        "protocol": {
+          "text": "Run protocol"
+        },
+        "simulation": {
+          "text": "Run simulation"
+        }
+      }
+    },
+    "card": {
+      "protocol": {
+        "parameters": {
+          "text": {
+            "driver": {
+              "name": "Driver name"
+            },
+            "population": {
+              "size": "Population size"
+            },
+            "total": {
+              "simulations": "Total simulations"
+            }
+          },
+          "title": {
+            "protocol": {
+              "parameters": "Protocol parameters"
+            }
+          }
+        }
+      },
+      "simulation": {
+        "parameters": {
+          "text": {
+            "alert": {
+              "simulation": {
+                "launched": "Simulation successfully launched"
+              }
+            },
+            "simulation": {
+              "name": "Simulation name"
+            },
+            "simulator": {
+              "name": "Simulator name"
+            }
+          },
+          "title": {
+            "simulation": {
+              "parameters": "Simulator parameters"
+            }
+          }
+        }
+      }
+    },
+    "iframe": {
+      "scenario": {
+        "results": {
+          "card": {
+            "title": "Results"
+          },
+          "iframe": {
+            "title": "Supply Chain results"
+          },
+          "text": {
+            "no": {
+              "result": "No results for this scenario."
+            }
+          }
+        }
+      }
+    },
+    "text": {
+      "contact": {
+        "get": {
+          "account": "Please contact the application administrator to activate your account."
+        }
+      },
+      "no": {
+        "permission": "You don't have permission to view this page."
+      }
+    }
+  },
+  "views": {
+    "common": {
+      "footer": {
+        "text": {
+          "poweredby": "Powered by CosmoTech"
+        }
+      },
+      "text": {
+        "loading": "Loading..."
+      }
+    },
+    "signin": {
+      "title": "Azure Sample Web Application"
```

Comment:
Is this title generated from a configuration file ? Or does the integrator have to configure it here (and at the other "title" key below too) ? I'm guessing this title parameter could be useful to be displayed elsewhere in the future (for ex in a "Help" section), the best option would be to set it in one place only

Reply:
Is this title generated from a configuration file ?  Nope
Or does the integrator have to configure it here (and at the other "title" key below too) ? Right now yes
 I'm guessing this title parameter could be useful to be displayed elsewhere in the future (for ex in a "Help" section), the best option would be to set it in one place only => It will be set info a config file one day in the project root folder ;)